src/Features/Lsif/GeneratorTest/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests.vbproj

```diff
@@ -0,0 +1,51 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
❔ Do we have any tests showing the output from this tool?

Reply:
Can you clarify what you had in mind?

Reply:
For a trivial assembly with a small number of types and members, can we verify the JSON output?

Reply:
I'll try to find a small project (maybe even just an empty file) and at least do one just so _something_ is validating textual output.

Reply:
Even in an trivial file the IDs aren't deterministic if we're doing parallel analysis... :-/

Reply:
Added some tests that are at least asserting that the serializer is working remotely correctly. This of course found a bug or two!